### DISCUSSION TITLE
why 404?

### DISCUSSION THREAD

**Discussion by z0z0r4** [community/community#12738] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/78744121/157998461-1661be8c-4200-40f5-89d8-1ffbc69e8617.png)
![image](https://user-images.githubusercontent.com/78744121/157998481-abdc55a6-7732-4952-bf45-5ebf3d995162.png)
![image](https://user-images.githubusercontent.com/78744121/157998494-edcaaf60-6bec-4059-95c4-e7a31175306c.png)
![image](https://user-images.githubusercontent.com/78744121/157998495-16078cee-873f-4a8c-96d6-7279241c0b06.png)

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
